spec/teacher_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rspec'
+require_relative '../classes/teacher'
+
+describe Teacher do
+  context 'Testing teacher class' do
```

Comment:
- [ ] Kindly add test for the `can_use_services?` method in the Teacher class

Reply:
Test for can_use_service in teacher class added